CHANGELOG.md

```diff
@@ -687,4 +687,10 @@ New Features:
 New Features:
 
 ### data-source/coralogix_webhook
-* Added support for searching by webhook name.
\ No newline at end of file
+* Added support for searching by webhook name.
+
+## Release 1.18.8
```

Comment:
Technically it will be 1.18.12 :)

Reply:
hmm. looks like the entry in `CHANGELOG.md` is missed for 1.18.8-11?

Reply:
😓